Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/AddKeyFlow.swift

```diff
@@ -123,12 +123,16 @@ class AddKeyFlow: UIFlow {
         assert(keyInfo != nil)
         assert(keyName != nil)
         let vc = factory.keyAdded(address: keyInfo!.address, name: keyName!, keyType: keyType) { [unowned self] in
-            stop(success: true)
+            didKeyAdded()
         }
 
         show(vc)
     }
 
+    func didKeyAdded() {
```

Comment:
This is grammatically incorrect, should be `didAddKey` (:+1) or `keyAdded`

Reply:
renamed to didDelegateKeySetup()